packages/template-retail-react-app/app/pages/home/index.jsx

```diff
@@ -48,6 +52,13 @@ import {
  */
 const Home = ({productSearchResult, isLoading}) => {
     const intl = useIntl()
+    const einstein = useEinstein()
+    const {pathname} = useLocation()
+
+    /**************** Einstein ****************/
+    useEffect(() => {
+        einstein.sendViewPage(pathname)
+    }, [pathname])
```

Comment:
What is the reason for using `pathname` as a source for this effect? One scenario I can think of is changing the locale or site if they are configured to be in path? but those typically use _hard_ refreshes to load.

Reply:
Pathname could be hard coded in the case of the home page since its path won't really be changing.
But I opted to keep the pattern here the same as the other pages where I am sending a viewPage activity.

Reply:
I think it would look less confusing to have pathname excluded in this instance. It also looks like you are using as many `[]` source values for pageViews as you are `[pathname]`. 

That being said, have you thought about using `Home.getTemplateName()` as the page name value sent for the page view.

Reply:
Correction.. I'm guessing that the pathname as more information in it that is useful for the using looking at the metrics.

Reply:
Not in this case actually so I can make the changes suggested here. The pathname at the moment is just `/` for the home page. This could change if we decide to send more information later on but for now I can change this to be [].

Reply:
Using Home.getTemplateName() is an interesting suggestion. In my local, I tried this and the activity sends `home` as the path rather than `/`.

For now, I'm going to keep `/` so that it is consistent for anyone who is reading the activities that `/` is the path rather than `/home`.

(I also tried Account.getTemplateName() on the account page but this contains less information than the pathname. In this case, getTemplateName() is always `account` rather than `/account/wishlist` or `/account/orders`)

Reply:
Actually, disregard the above. After a bit more testing, using pathname allows us to send the site and locale information. ie. the pathname for home can be `global/en-GB/` or `/global/es-MX/`